mne/utils/config.py

```diff
@@ -17,6 +17,10 @@
 from functools import partial
 from importlib import import_module
 from pathlib import Path
+from urllib.error import URLError
+from urllib.request import urlopen
+
+from packaging.version import parse
```

Comment:
I'm surprised this passed style checks, shouldn't it fail on import sorting grounds?

Reply:
This makes it a different import block